src/bashDebug.ts

```diff
@@ -285,7 +289,8 @@ export class BashDebugSession extends LoggingDebugSession {
 				const frameLine = parseInt(lineContent.substr(lineContent.lastIndexOf(" ")));
 
 				if ((process.platform === "win32")) {
-					frameSourcePath = this.getWindowsPathFromLinux(frameSourcePath);
+
+					frameSourcePath = frameSourcePath.startsWith("/") ? reverseWSLPath(frameSourcePath) : reverseWSLPath(`${this.workingDirectory}/${frameSourcePath}`);
 				}
```

Comment:
Needs testing/changes for non-windows OS

Reply:
Let me test on macOS.